report/src/problem_analysis/precence/our_choice.tex

```diff
@@ -1,24 +1,23 @@
-% We decided to use Bluetooth instead of object detection and environment sensing because:
-% - Stakeholder prefers radio technology
-% - Inconsitent accuracy (see longoAccurateOccupancyEstimation2019)
-% - Affected by unpredictable opening and closing of doors and windows (see longoAccurateOccupancyEstimation2019)
-% - Difficult to acquire
-% - Privacy concerns (cameras, object detection, etc.)
-% - Costly to setup and sensitive to light levels (video-based) 
-% - Sensitive to obstrutions in room
+\section{Choice of technology}
+In this section, we will compare the technologies presented in sections \ref{sec:presence_env} to \ref{sec:presence_radio} and decide the most fitting for our use case. 
+The requirements for this choice are written in listing \ref{lst:presence_eval_criteria}.
 
-% We decided to use Bluetooth instead of WiFi because:
-% - Specifically requested by stakeholder (often used in offices)
-% - Low power consumption
-% - Low cost
-% - Very cheap (see longoAccurateOccupancyEstimation2019)
-% - Works indoor and outdoor
-% - Easy to setup
-% - High availability
-% - In contrast to MAC address spoofing, Bluetooth devices can be identified by their Bluetooth address, which is unique to each device.
-% - Ideally, we should combine Wi-Fi and Bluetooth, however we do not need such high accuracy (see longoAccurateOccupancyEstimation2019), and it would be far more expensive in terms of hardware.
-% - We need local detection, however WiFi offers far greater range at the cost of accuracy, which is not needed for our application.
+We considered environmental-based technologies, as they are great at detecting presence in a room and are able to use multiple environmental sources, such as C02 level, light, humidity and temperature. 
+They also maintain the privacy of the occupants in the room. 
+A drawback of environmental-based technologies, as mentioned in section \ref{sec:presence_env}, is that they suffer from unpredictability when opening doors and windows. 
+Similarly, we also considered video-based object detection, and in section \ref{sec:presence_video}, we elaborated on the technology, citing that it has high accuracy but also a high cost of implementation and maintainance, is sensitive to lighting conditions, and raises privacy concerns.
+Because we require that the technology that we choose is both low cost, accurate and ensures privacy, this excludes these technologies as candidates for the project.
```

Comment:
Ref back to the requirements...
"As detailed in Section xxx, one requirement for a solution is that the technology used is both low cost, accurate and ensures privacy. This excludes video-based technologies as candidates for the solution.